Kitodo/src/main/java/org/kitodo/production/ldap/LdapUser.java

```diff
@@ -279,7 +279,7 @@ public static String toHexString(byte[] bytes) {
 
     @Override
     public Attributes getAttributes(String name) throws NamingException {
-        if (StringUtils.isBlank(name)) {
+        if (!StringUtils.isBlank(name)) {
```

Comment:
```suggestion
        if (StringUtils.isNotBlank(name)) {
```
Please use the dedicated, negated check instead of manually negating the positive check here, as it is more readable.